src/views/SystemIntake/Documents/UploadForm.tsx

```diff
@@ -169,7 +170,14 @@ const UploadForm = () => {
                       {t('technicalAssistance:errors.makeSelection')}
                     </ErrorMessage>
                   )}
-                  {['SOO_SOW', 'DRAFT_ICGE', 'OTHER'].map(val => (
+                  {[
+                    'SOO_SOW',
+                    'ACQUISITION_PLAN_OR_STRATEGY',
+                    'DRAFT_IGCE',
+                    'REQUEST_FOR_ADDITIONAL_FUNDING',
+                    'MEETING_MINUTES',
+                    'OTHER'
+                  ].map(val => (
```

Comment:
I also wonder if we could/should base this on an auto-generated type instead of just typing out the enums manually here 🤔 

Maybe not worth a refactor, but perhaps a `// TODO` comment to note a potential improvement!

Reply:
This can be achieved with `Object.values(SystemIntakeDocumentCommonType).map`, but it alphabetizes the options placing "Other" in the middle of the list.

Reply:
Good point! I think @patrickseguraoddball has done some stuff in the past on taking a generated type but allowing a custom order. If he can weigh in on it (and it's a simple approach) maybe we could try it, but otherwise I'm not too worried!

Reply:
Rather than map over those ordered strings, you'd want to map over translations of the mapped enum values.   This way you can order them as you want and keep them organized and strongly types to the translated values.

Something like this in your translation file:

```
export type DocumentOptionsType = Record<SystemIntakeDocumentCommonType, string>

const documentUploadOptions: Record<SystemIntakeDocumentCommonType, string> = {
ACQUISITION_PLAN_OR_STRATEGY: 'Text one',
DRAFT_IGCE: 'Text 2',
MEETING_MINUTES: 'Text 3',
OTHER: 'Other'
}
```

You then can import that from the `useTranslation` hook like so:
```
const documentUploadOptions: DocumentOptionsType = t(
    'documents:documentUploadOptions',
    { returnObjects: true }
  );
```

Reply:
Sometimes if you map over an object that does not return a string, we use this TS help util in place of Object.keys

Additionally we use this helper function to mapped over typed object to retain type:
```
// Util used to preserve type defintions when mapping over keys of object
// https://stackoverflow.com/questions/52856496/typescript-object-keys-return-string
export const getKeys = Object.keys as <T extends object>(
  obj: T
) => Array<keyof T>;
```

Reply:
Thanks Patrick!
Added here: [f96f4f4](https://github.com/CMS-Enterprise/easi-app/pull/2748/commits/f96f4f4ac0fb28237169316acafd8d76622db4fa)